src/pages/AccountIndexPage.re

```diff
@@ -216,6 +213,7 @@ let make = (~address, ~hashtag: Route.account_tab_t) => {
   let qrCode = () => {
     address->QRCode->OpenModal->dispatchModal;
   };
+  let addressTag = address->Address.toBech32->VerifyAccount.parseAddressName;
```

Comment:
1. Move `Address.toBech32` to `VerifiedAccount.re`
2. Change `parseAddressName` to `getNameOpt`.
3. Change VerifyAccount to VerifiedAccount.